gigascience/gigadb-website
# A11y - User create page

# Pull request for issue: #1387

This is a pull request for the following functionalities:

* Improve styles and accessibility of create user form http://gigadb.gigasciencejournal.com:9170/user/create

## How to test?

* Navigate to http://gigadb.gigasciencejournal.com:9170/user/create
* Run WAVE chrome extension and verify no errors
* Navigate through form via keyboard and check that focus states are visible

## How have functionalities been implemented?

* This work is branched from my integration branch (https://github.com/luistoptal/gigadb-website/tree/features-integration), so it has lots of changes that will be reduced once all previous PRs are merged to the upstream/develop
* Extensively updated the protected/views/user/_form.php
  * Tweak layout so that it looks more aligned and cleaner (less/current/modules/createuser-form.less)
  * Fix a11y issues reported by WAVE audit: color contrast, unlinked labels...
  * Used field components that incorporate a11y + bootstrap features
  * Updated the field components to work well with form-horizontal

## Any issues with implementation?

* Acceptance tests not passing at the moment, but this is caused by another branch, hopefully it will be fixed soon

## Any changes to automated tests?

None

## Any changes to documentation?

None

## Any technical debt repayment?

None

## Any improvements to CI/CD pipeline?

None
